Library/Homebrew/formula_creator.rb

```diff
@@ -242,7 +243,7 @@ def install
             system "meson", "compile", "-C", "build", "--verbose"
             system "meson", "install", "-C", "build"
         <% elsif @mode == :node %>
-            system "npm", "install", *std_npm_args
+            system "pnpm", "install", *std_npm_args
```

Comment:
The `*std_npm_args` are not a drop-in replacement here.

Reply:
I've had a bit of a play around trying to build a node-based project, and it looks like there could be some complexities with trying to build formula globally using `pnpm`. It likes to use its own global bin directory, so I need to do some more digging for how to configure this, but haven't been successful yet.